src/targets/cpu/lowering.cpp

```diff
@@ -376,13 +375,24 @@ struct cpu_apply
         // Apply these operators first so the inputs can be const folded
         for(auto it : iterator_for(*modl))
         {
-            if(it->name() == "pow")
+            // skip lowering if input has fp8 as one of the inputs since oneDNN doesn't have fp8
+            // supported yet
+            if(it->name() == "pow" and
+               (std::none_of(it->inputs().begin(), it->inputs().end(), [](const auto& i) {
+                   return i->get_shape().type() == migraphx::shape::fp8e4m3fnuz_type;
+               })))
```

Comment:
Can you just do `if(std::any_of(...)) continue;` like you did below?

Reply:
Done. 
But i don't see why that would make difference.

Reply:
It makes it easier to add operators to the first loop, and its consistent with the second loop.